Lorentz_model_comparison.ipynb

```diff
@@ -0,0 +1,606 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#Plot variable map of time step 50\n",
+    "array = ds[variable].isel(time= 2)\n",
+    "array.plot()\n",
+    "ds"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 161,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def dataframe_from_stationId(grdc_folder, station_id):\n",
+    "    ''' Search the grdc_folder for a file with name <station_id>.day,\n",
+    "    skip the header rows and return the dates and discharges in a dataframe'''\n",
+    "    \n",
+    "    filename = str(station_id) + \"_Q_Day.Cmd\"\n",
+    "    grdc_file = os.path.join(grdc_folder, filename)\n",
+    "\n",
+    "    grdc_data = pd.read_csv(grdc_file, skiprows=40, delimiter=';')\n",
+    "    \n",
+    "    \n",
+    "    \n",
+    "    grdc_data = grdc_data.rename(columns={grdc_data.columns[0]: 'date', grdc_data.columns[1]:'empty', grdc_data.columns[2]: 'discharge'})\n",
+    "    \n",
+    "    grdc_data = grdc_data.reset_index().set_index(pd.DatetimeIndex(grdc_data['date']))\n",
+    "    grdc_data = grdc_data[['date', 'discharge']]\n",
+    "    \n",
+    "    return grdc_data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 165,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import GRDC station data, set index\n",
+    "obs = dataframe_from_stationId('./validation/', 6435060)\n",
```

Comment:
Similar as model config+dataset, the validation file appears out of nowhere. Please provide instructions how to get it.